dpnp/backend/src/queue_sycl.cpp

```diff
@@ -34,6 +34,7 @@
 cl::sycl::queue* backend_sycl::queue = nullptr;
 #endif
 mkl_rng::mt19937* backend_sycl::rng_engine = nullptr;
+VSLStreamStatePtr backend_sycl::stream = NULL;
```

Comment:
It would be better to avoid global object for the stream (if it possible)

Reply:
We need it to save prev. state.